website/src/parsers/mdx/transformers/mdx/index.js

```diff
@@ -0,0 +1,28 @@
+import compileModule from '../../../utils/compileModule';
+import pkg from '@mdx-js/mdx/package.json';
+
+const ID = 'mdx';
+
+export default {
+  id: ID,
+  displayName: ID,
+  version: pkg.version,
+  homepage: 'https://mdxjs.com',
+
+  defaultParserID: 'MDXHAST',
```

Comment:
I'm not sure that I understand what this is for, so I'm not sure if I'm using it correctly. Could someone clarify?